src/pretix/presale/views/organizer.py

```diff
@@ -99,14 +99,19 @@ def get(self, request, *args, **kwargs):
     def get_queryset(self):
         query = Q(is_public=True) & Q(live=True)
         if "old" in self.request.GET:
-            query &= Q(Q(date_from__lte=now()) & Q(date_to__lte=now()))
+            qs = Event.objects.filter(
+                Q(organizer=self.request.organizer) & query
+            )
+            qs = qs.annotate(
+                min_to=Min('subevents__date_to')
+            ).filter(min_to__lte=now())
```

Comment:
You filter out normal events from the past here.

Reply:
You can run `py.test tests/presale/test_organizer_page.py` to test your changes. You should also add new tests for the behavior you are working on.